cdap-docs/application-templates/source/etl/templates/transformations/projection.rst

```diff
@@ -6,19 +6,56 @@
 Transformations: Projection
 ===========================
 
-.. rubric:: Description: Projection Transform: Drop, Rename, and Cast Fields
+.. rubric:: Description
 
-Projection transform that lets you drop, rename, and cast fields to a different type.
+A transform that can drop fields, rename fields, and cast fields to a different type.
+Fields are first dropped, then cast, then renamed.
+For example, suppose the transform is configured to drop field 'B' and rename field 'A' to 'B'.
+If the transform receives this input record::
 
-**Drop:** Comma separated list of fields to drop. For example: 'field1,field2,field3'.
+  +============================+
+  | field name | type | value  |
+  +============================+
+  | A          | int  | 10     |
+  | B          | int  | 20     |
+  +============================+
 
-**Rename:** List of fields to rename. This is a comma separated list of key-value pairs,
-where each pair is separated by a colon and specifies the input name and the output name.
+field 'B' will first be dropped::
+
+  +============================+
+  | field name | type | value  |
+  +============================+
+  | A          | int  | 10     |
+  +============================+
+
+and then field 'A' will be renamed to 'B'::
+
+  +============================+
+  | field name | type | value  |
+  +============================+
+  | B          | int  | 10     |
+  +============================+
+
+.. rubric:: Use Case
+
+The transform is used when you need to drop, rename, or change field types.
```

Comment:
The transform is used when you need to drop, change field types, or rename fields.